data/human/free worlds intro.txt

```diff
@@ -66,91 +75,85 @@ mission "Pact Recon 0"
+		dialog `You receive a message from Freya, the militia officer in charge of tracking pirate activity. "Hello, Captain <last>. I pray you're still interested in being a part of our Thousand Eyes program. We need some more intel on the empty systems in the middle of the Dirt Belt," she says. "Pay would be a bit better than last time, since these systems are a bit more out of the way. As with before, return to <destination> once you're finished."`
 	
 	on visit
 		dialog phrase "generic waypoint on visit"
 	on complete
 		payment 35000
-		dialog
-			`You transfer your sensor logs to Officer Winters. She thanks you for your work, and pays you <payment>. She says, "I'll let you know when we need your services again."`
+		dialog `You transfer your sensor logs to Freya. She thanks you for your work, and pays you <payment>. "I'll let you know when we need your services again," she says.`
 
 
 
 mission "Pact Recon 2"
 	landing
 	name "Pirate Reconnaissance"
 	description "Fly through the <waypoints> systems, then report to <destination> so the Southern Defense Pact can analyze your logs of pirate activity (if any)."
-	waypoint "Men"
-	waypoint "Antares"
-	waypoint "Shaula"
-	waypoint "Nunki"
 	source
-		attributes "south" "dirt belt" "rim" "near earth"
+		attributes "dirt belt" "near earth" "rim" "south"
 		not government "Pirate"
 	destination "Glaze"
+	waypoint "Antares"
+	waypoint "Men"
+	waypoint "Nunki"
+	waypoint "Shaula"
 	to offer
 		random < 40
-		not "event: war begins"
 		has "Pact Recon 1: done"
+		not "event: war begins"
 	
 	on offer
-		dialog
-			`You receive another message from Freya: "Hello Captain! I've got a more challenging and lucrative mission for you. We need someone to do a flyby of all four pirate systems here in the south, to gauge their strength. Again, no fighting is required. What do you say?"`
+		dialog `You receive another message from Freya: "Hello, Captain! I've got a more challenging and lucrative mission for you this time. We need someone to do a flyby of all four pirate systems here in the South, to gauge their strength. Again, no fighting is required, just fly in and out of the systems as fast as possible, and we're willing to pay much more than the last job given the danger involved. What do you say?"`
 	
 	on visit
 		dialog phrase "generic waypoint on visit"
 	on complete
-		payment 60000
-		dialog
-			`Freya Winters meets up with you in person soon after you land. "I hope you didn't have much trouble," she says, as she hands you a credit chip for <payment>. "You've been a great help to us. I'll be sure to contact you next time we need a scout."`
+		payment 70000
+		dialog `Freya Winters meets up with you in person soon after you land. "Glad to see you're still in one piece. I pray you didn't have much trouble," she says, as she hands you a credit chip for <payment>. "You've been a great help to us. I'll be sure to contact you next time we need a scout."`
```

Comment:
Her personal reappearance on Glaze then would feel more impactful here if you've previously dealt with an offsider on Glaze while she's been away.